### PR TITLE
Add Wagon.app latest

### PR DESCRIPTION
**Pull request by albertico**

### CODE REVIEW

File: Casks/wagon.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask :v1 => 'wagon' do
+  version :latest
+  sha256 :no_check
+
+  # amazonaws.com is the official download host per the vendor homepage
```
Comment (vitorgalvao) [inline Casks/wagon.rb]: Please remove the comment. It can only exist when it was confirmed by a maintainer.

**Comment by alebcay** [comment]
Warning: Download exists [behind a signup wall](https://app.wagonhq.com/download).

**Comment by vitorgalvao** [comment]
@alebcay Thank you for the notice (can never be to careful), but I had noticed as well, [hence why it’s being submitted to this repo](https://github.com/caskroom/homebrew-cask/pull/12450#discussion_r34067837).

**Comment by albertico** [comment]
@vitorgalvao Fixed!

**Comment by vitorgalvao** [comment]
Thank you. Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by albertico** [comment]
@vitorgalvao Done.  Commits squashed!

**Pull request merged**